Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -83,16 +80,21 @@ + (instancetype)BPRunnerForApp:(BPApp *)app
     return runner;
 }
 
-- (NSTask *)newTaskWithBundle:(BPBundle *)bundle andNumber:(NSUInteger)number andDevice:(NSString *)deviceID andCompletionBlock:(void (^)(NSTask *))block {
+- (NSTask *)newTaskWithBundle:(BPXCTestFile *)bundle
+                    andNumber:(NSUInteger)number
+                    andDevice:(NSString *)deviceID
+           andCompletionBlock:(void (^)(NSTask *))block {
     BPConfiguration *cfg = [self.config mutableCopy];
     assert(cfg);
-    cfg.testBundlePath = bundle.path;
-    cfg.testCasesToSkip = bundle.testsToSkip;
+    cfg.appBundlePath = bundle.testHostPath;
+    cfg.testBundlePath = bundle.testBundlePath;
+    if (bundle.UITargetAppPath) cfg.testRunnerAppPath = bundle.UITargetAppPath;
```

Comment:
I liked how you used the elvis operator `cfg.testRunnerAppPath = bundle.UITargetAppPath ?: nil;`